core-plugins/docs/JavaScript-transform.md

```diff
@@ -68,64 +70,62 @@ Currently supports ``KeyValueTable``.
 Example
 -------
 
-    {
-        "name": "JavaScript",
-        "type": "transform",
-        "properties": {
-            "script": "function transform(input, emitter, context) {
-                   var tax = input.subtotal * 0.0975;
-                   if (tax > 1000.0) {
-                     context.getMetrics().count("tax.above.1000", 1);
-                   }
-                   if (tax < 0.0) {
-                     context.getLogger().info("Received record with negative subtotal");
-                   }
-                   emitter.emit( {
-                     'subtotal': input.subtotal,
-                     'tax': tax,
-                     'total': input.subtotal + tax
-                   });
-                 }",
-            "schema": "{
-                \"type\":\"record\",
-                \"name\":\"expanded\",
-                \"fields\":[
-                    {\"name\":\"subtotal\",\"type\":\"double\"},
-                    {\"name\":\"tax\",\"type\":\"double\"},
-                    {\"name\":\"total\",\"type\":\"double\"}
-                ]
-            }",
-            "lookup": "{
-                \"tables\":{
-                    \"purchases\":{
-                        \"type\":\"DATASET\",
-                        \"datasetProperties\":{
-                            \"dataset_argument1\":\"foo\",
-                            \"dataset_argument2\":\"bar\"
-                        }
+```js
```

Comment:
this is not `json`?

Reply:
You are right it should be json. But having this a `js` helps in highlighting the code as part of the JSON + js also highlights json syntax as well.